Finalize generation script (#33)
redhat-ai-dev/ai-lab-helm-charts: charts/ai-software-templates/chatbot/README.md

This Helm Chart creates the following components:

### The Model Service

> By default the `chatbot-ai-sample` supports the `llama.cpp` inference server and related to the [ai-lab-recipes model server](https://github.com/containers/ai-lab-recipes/tree/main/model_servers/llamacpp_python).

Reviewer: ```suggestion
By default the `chatbot-ai-sample` uses the `llama.cpp` inference via the [ai-lab-recipes model server](https://github.com/containers/ai-lab-recipes/tree/main/model_servers/llamacpp_python).
```

Author: Fixed in [87d9c1e](https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/33/commits/87d9c1eb6630c67aa40b4c29ed17014d321a90d5)

> However, the usage of `vLLM` model services or existing model services is also supported:

Reviewer: ```suggestion
However, the use of `vLLM` to deploy model services, as well as the use of existing model services is also supported:
```

Author: Fixed in [87d9c1e](https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/33/commits/87d9c1eb6630c67aa40b4c29ed17014d321a90d5)
